doc/source/quickstart.rst

```diff
@@ -0,0 +1,422 @@
+KIWI Quickstart
+===============
+
+|Build Status| |Health|
+
```

Comment:
Does that work in the ReST source, meaning will it show the buildstatus and code health ?